frontend/src/app/components/wp-single-view-tabs/activity-panel/activity-tab.component.ts

```diff
@@ -26,20 +26,16 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {Component, Inject, OnDestroy} from '@angular/core';
-import {Transition} from '@uirouter/core';
-import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
-import {WorkPackageCacheService} from 'core-components/work-packages/work-package-cache.service';
+import {Component} from '@angular/core';
 import {WorkPackageResource} from 'core-app/modules/hal/resources/work-package-resource';
 import {ActivityPanelBaseController} from 'core-components/wp-single-view-tabs/activity-panel/activity-base.controller';
-import {WorkPackagesActivityService} from 'core-components/wp-single-view-tabs/activity-panel/wp-activity.service';
 import {AngularTrackingHelpers} from "core-components/angular/tracking-functions";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/components/wp-single-view-tabs/activity-panel/activity-tab.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>